File: Chapter-3/Python/Problem8.py
x,y = map(float,input().split())
if x == 0 and y == 0:
    print("The point lies on origin")
elif x != 0 and y == 0:
    print("The point lies on X-axis")
elif x == 0 and y != 0:
    print("The point lies on Y-axis")
else:
    print("The point lies on XY Plane");

File: Chapter-3/Java/Problem8.java
import java.util.Scanner;

public class Problem8 {
    public static void main(String[] args) {
        float x, y;
        Scanner sc = new Scanner(System.in);
        x = sc.nextFloat();
        y = sc.nextFloat();

        if (x == 0 && y == 0) {
            System.out.printf("The point lies on origin");
        } else if (x != 0 && y == 0) {
            System.out.printf("The point lies on X-axis");
        } else if (x == 0 && y != 0) {
            System.out.printf("The point lies on Y-axis");
        } else {
            System.out.printf("The point lies on XY plane");
        }
        sc.close();
    }
}
